examples/python/README.md

```diff
@@ -1,109 +1,96 @@
-# API Documentation
-
-The Python API documentation is
-available [here](https://scanbotsdk.github.io/documentation/barcode-scanner-sdk/linux/python-api/).
+# Scanbot SDK – Python on Linux
+This repository demonstrates how to use the [Scanbot SDK for Linux](https://scanbot.io) with Python.  
 
 ## NVidia Jetson
 
-> Scanbot SDK requires Jetpack 6.1, CUDA 12.6 and TensorRT 10.3 to run with GPU acceleration.
-
-* Install Python and OpenCV
+> Requires **Jetpack 6.1**, **CUDA 12.6**, and **TensorRT 10.3** for GPU acceleration.
 
+### 1. Install Python and OpenCV
 ```bash
 sudo apt install -y python3-venv nvidia-opencv
 ```
 
-* Optionally, install CUDA and TensorRT for GPU acceleration. Make sure that you're running a supported Jetpack version.
-
+### 2. (Optional) Install CUDA + TensorRT
+Make sure you are running a supported Jetpack version:
 ```bash
 sudo apt install -y nvidia-l4t-cuda libnvinfer10 libnvinfer-plugin10 libnvonnxparsers10
 ```
 
-* Create a virtual environment:
-
+### 3. Create and activate a virtual environment
 ```bash
 python3 -m venv .env --system-site-packages
 source .env/bin/activate
 pip install --upgrade pip setuptools wheel
 ```
 
-* Install the Scanbot SDK package
+### 4. Download and install the Scanbot SDK
+1. Go to [GitHub Releases](https://github.com/doo/scanbot-sdk-example-linux/releases)  
+2. Manually download the **Python package** for your platform (ARM64 or x86_64).  
+3. Install it with pip, for example:
 
 ```bash
-# specify the SDK-VERSION and SDK-API-TOKEN as specified along with the license file
-export SCANBOTSDK_VERSION=<SDK-VERSION>
-export SCANBOTSDK_API_TOKEN=<SDK-API-TOKEN>
-pip install https://scanbotsdk-deployment.s3.amazonaws.com/ScanbotSDK-Linux/${SCANBOTSDK_API_TOKEN}/scanbotsdk-${SCANBOTSDK_VERSION}-py3-none-linux_aarch64.whl
+pip install ./scanbotsdk-<VERSION>-py3-none-linux_aarch64.whl
 ```
 
-* Test that the scanbotsdk package has been installed correctly:
+or
 
 ```bash
-python -c "import scanbotsdk"
+pip install ./scanbotsdk-<VERSION>-py3-none-linux_x86_64.whl
 ```
 
-* You're now ready to run the examples.
-
-### Important performance notes
+### 5. Verify installation
+```bash
+python -c "import scanbotsdk"
+```
 
-* In order to achieve the best performance,
-  it is important to set GPU, CPU and memory frequencies to the maximum and accelerate the fan to prevent overheating.
-  This can be done using the `jetson_clocks` script:
+✅ You're ready to run the examples!
 
+### ⚡ Performance notes (Jetson)
+To avoid throttling, set max GPU/CPU/memory clocks:
```

Comment:
We need this note for all the wrappers which support live mode, i.e. for c as well